docs/topics/api/auth_internal.rst

```diff
@@ -20,57 +20,23 @@ Fetching the token
 ==================
 
 A fresh token, valid for 30 days, is automatically generated and added to the
-responses of the following endpoints:
+responses of the following endpoint:
 
-    * ``/api/v3/accounts/login/``
-    * ``/api/v3/accounts/register/``
+    * ``/api/v3/internal/accounts/login/start/``
```

Comment:
I think that's wrong. All `/login/start/` does is a redirect to the right URL for FxA, it can't set the token.

Reply:
So only `/api/v3/accounts/authenticate/` sets the token then?

Reply:
Looks like it, yes.